openstack/clustering/v1/policies/results.go

```diff
@@ -16,12 +16,14 @@ type Policy struct {
 	ID        string                 `json:"id"`
 	Name      string                 `json:"name"`
 	Project   string                 `json:"project"`
-	Spec      map[string]interface{} `json:"spec"`
+	Spec      Spec                   `json:"spec"`
 	Type      string                 `json:"type"`
 	UpdatedAt time.Time              `json:"-"`
 	User      string                 `json:"user"`
 }
 
+type PolicyFromCreate Policy
```

Comment:
Now that `Policy` can handle formats returned both in requests and responses, `PolicyFromCreate` should no longer needed. If it still is, let me know. The goal is to ultimately remove this type.

Reply:
Done